tests/conftest.py

```diff
@@ -18,7 +18,7 @@ def node(host, request):
     # because testinfra does not collect and provide ansible config passed in
     # from using --extra-vars
     ceph_stable_release = os.environ.get("CEPH_STABLE_RELEASE", "luminous")
-    node_type = ansible_vars["group_names"][0]
+    node_type = ansible_vars["group_names"]
```

Comment:
Just a small nit, but maybe this should be ``node_types`` now? Changing it to ``group_names`` might make even more sense actually.

Reply:
@andrewschoen true, that makes more sense. I've updated it